test/cypress/integration/worldpay-3ds-flex-ddc/payment-with-session-id.spec.js

```diff
@@ -0,0 +1,138 @@
+const cardPaymentStubs = require('../../utils/card-payment-stubs')
+
+describe('Worldpay 3ds flex card payment flow', () => {
+  const tokenId = 'be88a908-3b99-4254-9807-c855d53f6b2b'
+  const chargeId = 'ub8de8r5mh4pb49rgm1ismaqfv'
+  const sessionId = 'test session Id'
```

Comment:
Could these be named so that it’s clear which ones are posted back from the device data collection iframe?

Reply:
Sure, I've changed `sessionId` to `worldpaySessionId`